packages/core/src/components/axes/axis.ts

```diff
@@ -448,6 +522,12 @@ export class Axis extends Component {
 						datum,
 					}
 				);
+				if (!isTimeScaleType && axisOptions.scaleType && axisOptions.scaleType === "labels" && datum.length > truncationThreshold) {
```

Comment:
Can these checks be stored in a variable? They're fairly long, and we should use `Tools.getProperty()` to avoid the double checks (`axisOptions.scaleType && axisOptions.scaleType === "labels"`)

Reply:
Resolved. Thanks.